code/index.html

```diff
@@ -1,22 +1,153 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+
+<head>
+  <meta charset="utf-8" />
+  <title>W Magazine - Even Stranger Things</title>
+  <link rel="stylesheet" href="style.css" type="text/css" />
+  <link rel="icon" href="./img/logo.png" type="image/x-icon" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+</head>
+
+<body>
+
+  <!--HEADER-->
+  <header>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+    <img class="logo" src="./img/logo.png" alt="Logotype">
+    <div class="site-name">W Magazine</div>
+    <nav>
+      <a href="#">Journal</a>
+      <a href="#">Podcast</a>
+      <a href="#">In Brief</a>
+    </nav>
+  </header>
+
+  <!--BANNER-->
+
+  <div class="banner-container">
+    <p>&#128156; SUBSCRIBE</p>
+  </div>
+
+  <!-- THE BIG NEWS -->
+
+  <main>
+    <section class="big-news-container">
+      <img class="top" src="./img/BillyandMrsWheeler.jpg" alt="Billy and Mrs Wheeler" />
+      <div class="top-text">
+        <h3>TOP STORY</h3>
+        <h2>Billy and Mrs Wheeler in Court</h2>
+        <p>Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consectetura dipiscing elit, sed doipsum dolor sit
+          amet >>> </p>
+      </div>
+      <hr>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <!-- THE SMALLER NEWS  -->
+
+    <section class="small-news-container">
+      <div class="cards">
+        <img class="card-1" src="./img/Jazzercise.png" alt="Jazzercise image" />
```

Comment:
If you're not styling the images differently, you don't need different class names for them. Maybe you don't need a class name at all for the images? :)